src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailGlobalUIConfig.java

```diff
@@ -105,48 +97,13 @@ public UIComponent generateUIComponent() {
         final ConfigField mailSmtpMailExtension = new TextInputConfigField(EmailPropertyKeys.JAVAMAIL_MAILEXTENSION_KEY.getPropertyKey(), "Smtp Mail Extension", false, false, FieldGroup.ADVANCED);
         final ConfigField mailSmtpUserSet = new CheckboxConfigField(EmailPropertyKeys.JAVAMAIL_USERSET_KEY.getPropertyKey(), "Smtp User Set", false, false, FieldGroup.ADVANCED);
         final ConfigField mailSmtpNoopStrict = new CheckboxConfigField(EmailPropertyKeys.JAVAMAIL_NOOP_STRICT_KEY.getPropertyKey(), "Smtp NoOp Strict", false, false, FieldGroup.ADVANCED);
-        fields.add(mailSmtpPort);
-        fields.add(mailSmtpConnectionTimeout);
-        fields.add(mailSmtpTimeout);
-        fields.add(mailSmtpWriteTimeout);
-        fields.add(mailSmtpLocalhost);
-        fields.add(mailSmtpLocalAddress);
-        fields.add(mailSmtpLocalPort);
-        fields.add(mailSmtpEhlo);
-        fields.add(mailSmtpAuthMechanisms);
-        fields.add(mailSmtpAuthLoginDisable);
-        fields.add(mailSmtpAuthPlainDisable);
-        fields.add(mailSmtpAuthDigestMd5Disable);
-        fields.add(mailSmtpAuthNtlmDisable);
-        fields.add(mailSmtpAuthNtlmDomain);
-        fields.add(mailSmtpAuthNtlmFlags);
-        fields.add(mailSmtpAuthXoauth2Disable);
-        fields.add(mailSmtpSubmitter);
-        fields.add(mailSmtpDnsNotify);
-        fields.add(mailSmtpDnsRet);
-        fields.add(mailSmtpAllow8bitmime);
-        fields.add(mailSmtpSendPartial);
-        fields.add(mailSmtpSaslEnable);
-        fields.add(mailSmtpSaslMechanisms);
-        fields.add(mailSmtpSaslAuthorizationId);
-        fields.add(mailSmtpSaslRealm);
-        fields.add(mailSmtpSaslUseCanonicalHostname);
-        fields.add(mailSmtpQuitwait);
-        fields.add(mailSmtpReportSuccess);
-        fields.add(mailSmtpSslEnable);
-        fields.add(mailSmtpSslCheckServerIdentity);
-        fields.add(mailSmtpSslTrust);
-        fields.add(mailSmtpSslProtocols);
-        fields.add(mailSmtpSslCipherSuites);
-        fields.add(mailSmtpStartTlsEnable);
-        fields.add(mailSmtpStartTlsRequired);
-        fields.add(mailSmtpProxyHost);
-        fields.add(mailSmtpProxyPort);
-        fields.add(mailSmtpSocksHost);
-        fields.add(mailSmtpSocksPort);
-        fields.add(mailSmtpMailExtension);
-        fields.add(mailSmtpUserSet);
-        fields.add(mailSmtpNoopStrict);
+
+        final List fields = List.of(mailSmtpHost, mailSmtpFrom, mailSmtpAuth, mailSmtpUser, mailSmtpPassword, mailSmtpPort, mailSmtpConnectionTimeout, mailSmtpTimeout, mailSmtpWriteTimeout, mailSmtpLocalhost, mailSmtpLocalAddress,
```

Comment:
I like the use of List.of rather than dozens of adds, but given the number of fields, it might be worth it to have them on new lines.